k8sutils/pkg/utils/config_util.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/odigos-io/odigos/common"
+	"github.com/odigos-io/odigos/common/consts"
+	"github.com/odigos-io/odigos/k8sutils/pkg/env"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+)
+
+func GetCurrentConfig(ctx context.Context, k8sClient client.Client) (common.OdigosConfiguration, error) {
```

Comment:
nit: "Config" can mean few things, perhaps we can rename it `GetCurrentOdigosConfig` so that it communicate what is being returned in the function name